### PR TITLE
feat(highlights): Generate text item from highlight package

### PR DESCRIPTION
**Pull request by nidzix**
SD-1709

### CODE REVIEW

**Comment by ioanpocol** [comment]
:+1:

**Comment by ioanpocol** [comment]
There is a small flake8 issue:
./apps/highlights/generate.py:14:9: F841 local variable 'ids' is assigned to but never used

File: client/app/scripts/superdesk-highlights/highlights.js [hunk 1]
```diff
@@ -298,6 +298,10 @@
             type: 'http',
             backend: {rel: 'marked_for_highlights'}
         });
+        apiProvider.api('generate_highlights', {
```
Comment (petrjasek) [inline client/app/scripts/superdesk-highlights/highlights.js]: there is no need for this if you use `api.save`

**Comment by petrjasek** [comment]
fixed the flake8 issue on branch

**Comment by mugurrus** [comment]
:+1:

**Pull request merged**